presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcPageSink.java

```diff
@@ -163,6 +167,12 @@ else if (DATE.equals(type)) {
             long localMillis = getInstanceUTC().getZone().getMillisKeepLocal(DateTimeZone.getDefault(), utcMillis);
             statement.setDate(parameter, new Date(localMillis));
         }
+        else if (TIME.equals(type)) {
+            statement.setTime(parameter, new Time(type.getLong(block, position)));
+        }
```

Comment:
Based on my read of the code - `time` is stored in terms of UTC time and not local time and hence we may have to convert this to UTC time before writing the value back.

https://github.com/prestodb/presto/blob/master/presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/StandardReadMappings.java#L152 -> this is how we read the time value

https://github.com/prestodb/presto/blob/master/presto-common/src/main/java/com/facebook/presto/common/type/SqlTime.java#L98

@highker - what is your opinion here?

Reply:
Yes, it's better to be consistent with `DATE` branch